src/Review.jsx

```diff
@@ -0,0 +1,17 @@
+const Review = ({ review }) => {
+  console.log(review);
+  return (
+    <div className="displayReviewsBox">
+      <p className="owner">{review.owner}</p>
+      <p className="date">Date: {review.created_at}</p>
+      <p className="title">{review.title}</p>
+      <img className="image" src={review.review_img_url} />
+      <p className="category">Category: {review.category}</p>
+      <p className="designer">Designer: {review.designer}</p>
+      <p className="review">{review.review_body}</p>
```

Comment:
I would remove the body from here. The list of reviews shouldn't really show the body text as it will really bloat the page!